_includes/devdoc/guide_dash_features.md

```diff
@@ -110,6 +109,100 @@ had already been confirmed to a block depth of 5 in the blockchain.
 
 {% endautocrossref %}
 
+#### LLMQ InstantSend
+{% include helpers/subhead-links.md %}
+
+{% autocrossref %}
+
+The introduction of Long-Living Masternode Quorums in Dash Core 0.14 provides
+a foundation to further scale InstantSend. LLMQ-based InstantSend removes a
+number of previously required limitations and simplifies the process by decreasing
+the number of P2P messages clients must use.
+
+During the evaluation and transition from standard InstantSend to LLMQ-based
+InstantSend, Sporks 2 (`SPORK_2_INSTANTSEND_ENABLED`) and 20 (`SPORK_20_INSTANTSEND_LLMQ_BASED`)
+will both be used. Spork 2 enables or disables the entire InstantSend feature,
+while spork 20 determines which of the two InstantSend mechanisms is active when
+InstantSend is enabled.
+
+There are still some limitations on LLMQ-based InstantSend transactions:
+
+* Transaction inputs must either:
+  * Be in a block that has a ChainLock
+  * Have at least the number confirmations (block depth) indicated by the table below
+
+  | **Network** | **Confirmations Required** |
+  |---------|--------------|
+  | Mainnet | 6 Blocks |
+  | Testnet | 2 Blocks |
+  | Regtest | 2 Blocks |
+  | Devnet  | 2 Blocks |
+
+
+Improvements from the old InstantSend system:
+
+* Changed: Transactions can be chained if the inputs are from transactions that are also locked
+* Changed: InstantSend locks are attempted for all transactions (`tx` messages) - no need to request it via the special message (`ix` message)
+* Changed: Only need to receive a single `islock` message - no need to track votes (`txlvote` messages) for each input
+* Removed: Limit on number of inputs
```

Comment:
Don't love the change of prefix, I would prefer consistent prefix but because I ended up not reading it by the fourth line so didn't see it changed and was confused :confused:

Reply:
Yes, it is a little awkward. What do you think about removing the prefix and just doing this:

* Transactions can now be chained if the inputs are from transactions that are also locked
* InstantSend locks are attempted for all transactions (`tx` messages) - no need to request it via the special message (`ix` message)
* Only need to receive a single `islock` message - no need to track votes (`txlvote` messages) for each input
* Limit on number of inputs removed
* Limit on transaction value removed
* Timeout for locking removed - transaction locks will only be removed once the transaction is confirmed in a ChainLocked block
* Custom InstantSend fee remove

Reply:
I like it :)

Reply:
I actually changed it to be a table - https://github.com/dash-docs/dash-docs/blob/204d9def1af48b9a7d1b25ce612709aa41bb391c/_includes/devdoc/guide_dash_features.md#llmq-instantsend